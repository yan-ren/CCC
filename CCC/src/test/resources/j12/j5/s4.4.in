5
1 2 5 4 3
5
5 2 1 3 4
6
6 5 3 4 2 1
6
5 4 6 1 2 3
6
4 3 5 6 2 1
0
