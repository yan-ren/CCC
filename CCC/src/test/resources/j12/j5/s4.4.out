18
22
38
28
26
